james_kelley/range.rb

```diff
@@ -0,0 +1,11 @@
+x = (1..30)
+
+puts "Class Name: #{x.class}"
+
+puts "Range includes 15!" if x.include? 15
+
+puts "Last number is " + x.last.to_s
```

Comment:
Try and use interpolation so: "Last number is #{x.last}" it auto to_s it as well.